### ISSUE TITLE
[BUG/REQUEST] member with id in guild

### ISSUE THREAD

**Issue by vitusverden** [SkyCraft78/DiSky#4]
**Name:** `member with id in guild`

**Potential New Syntaxes:** [disky] member with id %string% in [guild] %guild%

**Other syntaxes affected (if any):** `member/user with id

**In-dept description of the new features:**
```
Member with ID doesn't really work, as it needs a guild to know where the member is. This is a request to fix that.
```

**Comment by SkyCraft78** [comment]
Hey!
Nope, the bot can found a guild-based entity (like channel, role, or event **member**) without a specific Guild.
The only one is message, where you need to specify a channel there.
You can normally use `member with id "ID"` to get a member via its ID, but say me if it doesn't work.

**Comment by vitusverden** [comment]
Member with id "ID" does not work: https://discord.com/channels/818182471140114432/818182471887749126/828367231900581929
https://discord.com/channels/818182471140114432/818182471677247527/828361315931258912

**Comment by WeeskyBDW** [comment]
Hey
Can you give an example code and the error ?

Thx

**Comment by SkyCraft78** [comment]
Fixed in f8d8177 & a68bf51
